ocaml/typing/typeclass.ml

```diff
@@ -430,6 +430,7 @@ and class_type_aux env virt self_scope scty =
       cltyp (Tcty_signature clsig) typ
 
   | Pcty_arrow (l, sty, scty) ->
+      let l = transl_label l in
```

Comment:
What if there's a `[%src_pos]` here? This would happen in this scenario (which could occur in a `sig`):

```
class c : src_pos:[%src_pos] -> object end
```

This will probably work out OK, actually, because the call to `transl_simple_type` below would complain about the `[%src_pos]`. But it's definitely a bit wonky.